mne/minimum_norm/tests/test_inverse.py

```diff
@@ -891,4 +892,40 @@ def test_inverse_ctf_comp():
         apply_inverse_raw(raw, inv, 1. / 9.)
 
 
+def _check_delayed_data(inst, delayed):
+    """Check whether data is represented as kernel or not."""
+    if delayed:
+        assert isinstance(inst._kernel, np.ndarray)
+        assert isinstance(inst._sens_data, np.ndarray)
+        assert inst._data is None
+        assert not inst._kernel_removed
+    else:
+        assert inst._kernel is None
+        assert inst._sens_data is None
+        assert isinstance(inst._data, np.ndarray)
+
+
+@testing.requires_testing_data
+@pytest.mark.parametrize('pick_ori, inv_file', [['normal', fname_inv],
+                                                ['vector', fname_inv],
+                                                ['vector', fname_vol_inv]])
```

Comment:
@massich
I can't test for VolSourceEstimates here (pick_ori="normal", inv_file = fname_vol_inv), because of an unrelated error that results from inverse_operator["orient_prior"] being None for this particular inverse operator.

I checked it, however, by manually replacing this variable, and it works. But this would be an ugly hack. Unfortunately, there's no alternative volume inverse operator, and it seems a bit far fetched to me to construct everything from the forward solution on.
So, is it okay to keep it like this? Or is there an alternative I didn't notice?